tests/ui-testing/pages/pipelinePage.js

```
@@ -0,0 +1,273 @@
+    await this.confirmDeleteButton.click();
+  }
+
+  // Method to verify deletion success message
+  async verifyPipelineDeleted() {
+    await this.deletionSuccessMessage.click();
+  }
+
+
+  async selectStream() {
+    await this.queryButton.click();
+  }
```

Biome

[error] Duplicate class member name "selectStream"



(lint/suspicious/noDuplicateClassMembers)